### PR TITLE
Tweak: Prompt the user permission to allow unfiltered file uploads in Import Template flow [ED-5279]

### PR DESCRIPTION
**Pull request by udidol**
## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Prompt the user permission to allow unfiltered file uploads in Import Template flow

### CODE REVIEW

File: includes/template-library/manager.php [hunk 1]
```diff
@@ -450,9 +450,16 @@ public function direct_import_template() {
 	 * @return mixed Whether the export succeeded or failed.
 	 */
 	public function import_template( array $data ) {
+		// If the template is a JSON file, allow uploading it.
+		add_filter( 'elementor/allow-file-type/json', [ $this, 'enable_json_template_upload' ] );
+		add_filter( 'elementor/files/allow_unfiltered_upload', [ $this, 'enable_json_template_upload' ] );
```
Comment (kobizz) [inline includes/template-library/manager.php:455]: Remove this filter and handle deprecated methods

File: includes/template-library/manager.php [hunk 2]
```diff
@@ -450,9 +450,16 @@ public function direct_import_template() {
 	 * @return mixed Whether the export succeeded or failed.
 	 */
 	public function import_template( array $data ) {
+		// If the template is a JSON file, allow uploading it.
+		add_filter( 'elementor/allow-file-type/json', [ $this, 'enable_json_template_upload' ] );
```
Comment (kobizz) [inline includes/template-library/manager.php:454]:
```suggestion
		add_filter( 'elementor/files/allow-file-type/json', [ $this, 'enable_json_template_upload' ] );
```

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/template-library/manager.php (hunk 1 above)
Comment (udidol) [reply includes/template-library/manager.php:455]: The filter cannot be removed, as it is necessary for different usages in the product.
The specific file type filter is necessary for JSON files in order to enable importing single templates.
@kobizz

File: includes/template-library/manager.php (hunk 2 above)
Comment (udidol) [reply includes/template-library/manager.php:454]: Done

File: includes/template-library/manager.php (hunk 1 above)
Comment (udidol) [reply includes/template-library/manager.php:455]: Handling deprecated methods - done.

**Pull request merged**